metallb/run_e2e.sh

```diff
@@ -20,6 +20,12 @@ sudo firewall-cmd --zone=libvirt --permanent --add-port=179/tcp
 sudo firewall-cmd --zone=libvirt --add-port=179/tcp
 sudo firewall-cmd --zone=libvirt --permanent --add-port=180/tcp
 sudo firewall-cmd --zone=libvirt --add-port=180/tcp
+sudo firewall-cmd --zone=libvirt --permanent --add-port=3784/tcp
```

Comment:
BFD protocol is over UDP: https://datatracker.ietf.org/doc/html/rfc5881#section-4

Reply:
Ah dang, I think I even noticed that and that got lost in copy-paste land. Thanks for catching this.